musicbrainz/MusicBrainzCLI/server.js

```diff
@@ -0,0 +1,69 @@
+var request = require("request");
+var fs = require('fs');
+var getReleases = require("./getReleases.js");
+var getWorks = require("./getWorks.js");
+var getArtists = require("./getArtists.js");
+var intervalId = null;
+
+var obj = {
+    table: []
+};
+
+var date = 1620;
+
+
+
+intervalId = setInterval(function () {
+    // once all requests for the year 1620 until 1820 are executed, convert the outputobject into json and write it to file
+    if (date == 1623) {
+
+        clearInterval(intervalId);
+            
+        arrayofObj = obj.table;
+        var finalArray = arrayofObj.map(function (artistai) {
+            return artistai.id;
+        });
+
+        getReleases.getReleases(finalArray);
+        getWorks.getWorks(finalArray);
+        getArtists.getArtists(finalArray);
+
```

Comment:
that's where the process.send would need to be. However, since we call the scripts differently due to changes in the cli.js, you can just leave that for now and I'll implement the changes later on